tests/extmod/vfs_blockdev.py.exp

```diff
@@ -1,5 +1,5 @@
 test <class 'VfsFat'>
-(512, 512, 16, 16, 16, 0, 0, 0, 0, 255)
+(512, 512, 44, 44, 44, 0, 0, 0, 0, 255)
```

Comment:
Will these change given my change from 64 to 128 above?

Reply:
Yes. I'll update them.